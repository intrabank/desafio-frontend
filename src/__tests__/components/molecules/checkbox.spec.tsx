import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Checkbox from '../../../components/molecules/checkbox';

describe('Checkbox', () => {
	it('should render checkbox component', () => {
		const tree = renderer
			.create(<Checkbox text='this is an checkbox text' />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
