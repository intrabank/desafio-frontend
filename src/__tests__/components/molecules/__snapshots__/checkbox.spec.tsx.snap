// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should render checkbox component 1`] = `
<div
  className="c-faRsTb"
>
  <input
    aria-label="this is an checkbox text"
    role="checkbox"
    type="checkbox"
  />
  <span
    className="c-eFhfvJ c-eFhfvJ-fPSeYF-color-white c-eFhfvJ-TkcIS-size-sm c-eFhfvJ-jzkLDL-weight-light"
  >
    this is an checkbox text
  </span>
</div>
`;
